ocaml/util/xapi_version.ml

```diff
@@ -20,9 +20,11 @@ let hostname = "localhost"
 
 let date = Build_info.date
 
-let xapi_version_major, xapi_version_minor =
+let xapi_version_major, xapi_version_minor, xapi_version_rel =
```

Comment:
I think `micro` and `%d`.

Reply:
With `%d` we would miss extra information also present: `22.5.0.18.gb60d592f7.2.2.g014ff5d.xs8` - this is currently captured.

Reply:
OK, on a release RPM build, this would only be the micro version, but on a dev build it will have further stuff on it (not the release number though).